### PR TITLE
Add Civo navigate material, update content

### PR DESCRIPTION
**Pull request by technosophos**
Updates to the text to bring up to Spin v2 and Civo cloud

### CODE REVIEW

File: wasm-and-containers/workshop/03-deploy-spin-to-k8s.md [hunk 1]
```diff
@@ -30,16 +30,25 @@ In addition to the requirements from the previous sections, you'll need the foll
...
 
 ## 1. Create and Configure the k3d cluster
 
+> See [The Spin Kubernetes docs](https://developer.fermyon.com/spin/v2/kubernetes) for more information on configuring various Kubernetes flavors.
+
 The first step is to create a k3d cluster with the wasm shims installed:
 
 ```bash
-k3d cluster create wasm-cluster --image ghcr.io/deislabs/containerd-wasm-shims/examples/k3d:v0.9.2 -p "8081:80@loadbalancer" --agents 2
+k3d cluster create wasm-cluster --image ghcr.io/deislabs/containerd-wasm-shims/examples/k3d:v0.10.0 -p "8081:80@loadbalancer" --agents 2 --registry-create mycluster-registry:12345
```
Comment (kate-goldenring) [inline wasm-and-containers/workshop/03-deploy-spin-to-k8s.md:48]: Should we bump to the latest image? https://github.com/deislabs/containerd-wasm-shims/blob/main/containerd-shim-spin/quickstart.md#start-and-configure-a-k3d-cluster
```suggestion
k3d cluster create wasm-cluster --image ghcr.io/deislabs/containerd-wasm-shims/examples/k3d:v0.11.1 -p "8081:80@loadbalancer" --agents 2 --registry-create mycluster-registry:12345
```

File: wasm-and-containers/README.md [hunk 2]
```diff
@@ -5,13 +5,15 @@ This workshops contains:
...
 
 **Last Updated**
-This workshop was last updated for KubeCon North America 2023
+This workshop was last updated for Civo Navigate Austin, 2024.
 
 ## Workshop Modules
 
 - [0. Setup](./workshop/00-setup.md)
 - [1. Getting started with Spin and WebAssembly](./workshop/01-spin-getting-started.md)
 - [2. Run your first WebAssembly application in a container](./workshop/02-running-in-a-container.md)
```
Comment (kate-goldenring) [inline wasm-and-containers/README.md:14]: I believe the container section will need to be updated to use the `v2` shim:
`docker run -d --platform=wasi/wasm --runtime=io.containerd.spin.v2 -p 3000:80`

**Review by kate-goldenring — APPROVED** [review]
LGTM. Only other section that will need updates in the container section for `v2`

File: wasm-and-containers/README.md (hunk 2 above)
Comment (technosophos) [reply wasm-and-containers/README.md:14]: @jpflueger just fixed it. Thanks!

**Pull request merged**